native/cocos/scene/Model.cpp

```diff
@@ -510,6 +510,50 @@ void Model::updateLocalShadowBias() {
     _localDataUpdated = true;
 }
 
+void Model::updateReflctionProbeCubemap(TextureCube *texture) {
+    _localDataUpdated = true;
+    if (texture == nullptr) {
+        texture = BuiltinResMgr::getInstance()->get<TextureCube>(ccstd::string("default-cube-texture"));
+    }
+    gfx::Texture *gfxTexture = texture->getGFXTexture();
+    if (gfxTexture) {
+        auto *sampler = _device->getSampler(texture->getSamplerInfo());
+        for (SubModel *subModel : _subModels) {
+            gfx::DescriptorSet *descriptorSet = subModel->getDescriptorSet();
+            descriptorSet->bindTexture(pipeline::REFLECTIONPROBECUBEMAP::BINDING, gfxTexture);
+            descriptorSet->bindSampler(pipeline::REFLECTIONPROBECUBEMAP::BINDING, sampler);
+            descriptorSet->update();
+        }
+    }
+}
+void Model::updateReflctionProbePlanarMap(gfx::Texture *texture) {
+    _localDataUpdated = true;
+
+    IntrusivePtr<gfx::Texture> bindingTexture;
+    if (texture == nullptr) {
+        bindingTexture = BuiltinResMgr::getInstance()->get<Texture2D>(ccstd::string("empty-texture"))->getGFXTexture();
+    } else {
+        bindingTexture = texture;
+    }
```

Comment:
- i think use raw pointer for bindingTexture is enough
- the codes can be simplified like
```c++
gfx::Texture *bindingTexture = texture;
if (!bindingTexture) {
    bindingTexture = BuiltinResMgr::getInstance()->get<Texture2D>(ccstd::string("empty-texture"))->getGFXTexture();
}
```